proto/server/configEvtHandler.go

```diff
@@ -163,6 +164,17 @@ func configHandler(configMsgChan chan *configmodels.ConfigMessage, configReceive
 	}
 }
 
+func handleSubscriberPost(configMsg *configmodels.ConfigMessage) {
```

Comment:
Should we also add the `get` method(s) for this? The other `handle` functions have `get` functions.

Reply:
That is because the "subscriber" is handled a bit differently in the code.
The subscriber object does not exist as such, when a call is made to `get subscriber by ID`, the object is created by fetching data from different collections, and put into the `SubsData` object.
When we make a `post call to create the subscriber`, we don't really create the subscriber, we post a message with the `ueId` to `configChannel` and then update the `AuthenticationSubscription` data collection. Only when the device group and network slice are created the rest of the data required to build the `SubsData` object is added.
In this PR my aim is to save the `ueId` to the `AmData` even before creating the device group and network slice so we do get a list of the subscribers when we call `GetSubscribers`.
Having that said, based on my understanding of how subscribers are handled in the code, adding a `getSubscriber` here would be misleading, as we can't be sure of having all ghe bits needed ti build the `SubsData` right after a `subscriber post`, all we have for sure is the `ueId`.
The handler of the `get request for a subscriber` is in `api_sub_config.go` file as `GetSubscriberByID` and `GetSubscribers`.

Please let me know if I have answered the question or not, or if I'm missing anything here :)